packages/flutter/lib/src/widgets/mimic.dart

```diff
@@ -59,10 +69,18 @@ class MimicOverlayEntry {
     return _performance.play();
   }
 
+  /// Cause the overlay entry to rebuild during the next pipeline flush.
+  ///
+  /// You need to call this function if you rebuild the widget that this entry
+  /// is mimicking in order for the overlay entry to pick up the changes that
+  /// you've made to the [Mimicable].
```

Comment:
That's too late.  You want to call this function from an event handler or an animation tick.  Something before the build process starts.  That way you get into the same frame.